certification/internal/operatorsdk/operatorsdk.go

```diff
@@ -1,29 +1,32 @@
-package engine
+package operatorsdk
 
 import (
 	"bytes"
+	"context"
 	"encoding/json"
 	"fmt"
 	"os"
 	"os/exec"
 	"strings"
 
 	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification/artifacts"
-	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification/internal/cli"
 	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification/runtime"
 	log "github.com/sirupsen/logrus"
 )
 
-func NewOperatorSdkEngine(userProvidedScorecardImage string) *cli.OperatorSdkEngine {
-	var engine cli.OperatorSdkEngine = operatorSdkEngine{scorecardImage: userProvidedScorecardImage}
+func New(userProvidedScorecardImage string, cmdContext execContext) *operatorSdk {
+	engine := operatorSdk{scorecardImage: userProvidedScorecardImage, cmdContext: cmdContext}
 	return &engine
 }
 
-type operatorSdkEngine struct {
+type operatorSdk struct {
 	scorecardImage string
+	cmdContext     execContext
 }
 
-func (o operatorSdkEngine) Scorecard(image string, opts cli.OperatorSdkScorecardOptions) (*cli.OperatorSdkScorecardReport, error) {
+type execContext = func(name string, arg ...string) *exec.Cmd
```

Comment:
I know it's not public, but let's add a comment here so we understand what's happening without having to find uses of this type.

Reply:
Done.